FULL_README.md

```
@@ -104,6 +116,9 @@
     * [-h/--help](#-h--help)
 
 - [All command line options](#All-command-line-options)
+- [Scope](#Scope)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid
Expected: `#scope`; Actual: `#Scope`

(MD051, link-fragments)